dot/core/service.go

```diff
@@ -251,6 +254,11 @@ func (s *Service) handleBlocks(ctx context.Context) {
 				log.Warn("failed to store storage trie in database", "error", err)
 			}
 
+			err = s.handleChainReorg(prev, block.Header.Hash())
+			if err != nil {
+				log.Warn("failed to readd transactions to chain upon re-org", "error", err)
```

Comment:
readd -> read

Reply:
this should say readd, I'll change it to re-add